logic/transaction.js

```diff
@@ -591,7 +598,11 @@ Transaction.prototype.verify = function (trs, sender, requester, cb) {
 			return setImmediate(cb, err);
 		} else {
 			// Check for already confirmed transaction
-			return self.checkConfirmed(trs, cb);
+			if(checkPersistent) {
+				return self.checkConfirmed(trs, cb);
```

Comment:
please correct name of function `checkConfirmed` - it's misleading.

Reply:
Its used on more than one places in code with same name. Not seems right to rename it in the current scope.